contents/teams/replay/objectives.mdx

```diff
@@ -1,24 +1,18 @@
-### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
+### 🎯 Goal 1: People think of PostHog as a mobile solution
 
-Why? 
+* Mobile replay for React Native
+* Lots of content on how to use PostHog on Mobile
```

Comment:
It feels like we're light on product / engineering work for this goal unless we're planning to lead the content over marketing & docs.

Maybe this detail is ok to be left to the Mega issue but some ideas of potential line items we could add here:
- 5 customer feedback interviews
- Launch & charge for mobile replay

Reply:
> we're planning to lead the content over marketing & docs.

my goal here would be to pair up with them, provide the technical bits, and help out, but they would help with the writing, making it nice.
@andyvan-ph would that be something your team can help with?

Reply:
I agree, we can add that "Launch & charge for mobile replay" is part of our goal, and since any improvement is done in 2 and soon 3 platforms (hopefully the wrapper approach alone is enough), the work adds up, I also have a couple of weeks OOO this Q.
Most likely we will add ad-hoc work such as PoE in the Mobile SDKs, etc.